src/Moq/SetupCollection.cs

```diff
@@ -32,7 +32,7 @@ public void Add(Setup setup)
 		{
 			lock (this.setups)
 			{
-				if (setup.Method.LooksLikeEventAttach() || setup.Method.LooksLikeEventDetach())
+				if (setup.Method.IsEventAddAccessor() || setup.Method.IsEventRemoveAccessor())
```

Comment:
For consistency, add a `.IsEventAccessor()` check method.

Reply:
Left as is for now, as the new method would be used in only two places, and not introduce any opportunities for optimization.